StreamChatUITestsAppUITests/Robots/UserRobot+Asserts.swift

```diff
@@ -615,24 +649,53 @@ extension UserRobot {
         XCTAssertTrue(link.isHittable, "Link itself is not clickable")
         return self
     }
+    
+    @discardableResult
+    func waitForMessageVisibility(at messageCellIndex: Int) -> Self {
+        _ = messageCell(withIndex: messageCellIndex).waitForHitPoint()
+        return self
+    }
 }
 
 // MARK: Quoted Messages
 extension UserRobot {
 
     @discardableResult
     func assertQuotedMessage(
-        replyText: String,
+        replyText: String = "",
```

Comment:
Why is this empty string by default ? 🤔

Reply:
To use the same assert method when the message has no text (e.g. when you send an attachment without text)

Reply:
Maybe better to add a comment? 🤔